### PR TITLE
Unfrobulate button

### PR DESCRIPTION
**Pull request by whit537**
I thought I made a ticket for this but I'm not finding it now. Buttons are frobulated on elsewheres:

screen shot 2015-02-10 at 10 11 32 pm

### CODE REVIEW

**Comment by whit537** [comment]
After unfrobulation:

screen shot 2015-02-10 at 10 20 52 pm

**Comment by whit537** [comment]
Unauthed still frobulated:

screen shot 2015-02-10 at 10 21 36 pm

**Comment by whit537** [comment]
Okay, ready for review.

**Comment by whit537** [comment]
screen shot 2015-02-10 at 11 10 04 pm

**Comment by Changaco** [comment]
The ticket is #3096.

File: tests/py/test_elsewhere.py [hunk 1]
```diff
@@ -103,12 +103,23 @@ def test_user_pages(self):
...
-                            )
+                             )
             platform.get_user_info = lambda *a: alice
             response = self.client.GET('/on/%s/alice/' % platform.name)
             assert response.code == 200
             assert 'has not joined' in response.body.decode('utf8')
 
+    def test_user_name_is_in_button(self):
+        self.make_participant('bob', claimed_time='now')
+        UserInfo( platform='twitter', user_id='0', user_name='alice', is_team=False)
```
Comment (Changaco) [inline tests/py/test_elsewhere.py]: That's a noop, you're not actually doing anything with the `UserInfo`.

**Comment by Changaco** [comment]
I've dropped the test fixture changes and added a couple of commits. Ready for review.

**Pull request merged**